.circleci/config.yml

```diff
@@ -351,3 +367,34 @@ jobs:
           path: test-results/junit
       - store_artifacts:
           path: test-results
+
+  build-package:
+    executor: golang
+    parameters:
+      package_type:
+        type: enum
+        enum: ["deb", "rpm"]
+    steps:
+      - checkout
+      - attach_to_workspace
+      - run:
+          name: Install fpm and dependencies
+          command: |
+            sudo apt-get update
+            sudo apt-get install -y ruby ruby-dev rubygems build-essential rpm
+            sudo gem install --no-document fpm -v 1.11.0
```

Comment:
Instead of all this, can this job use docker executor with an FPM image (such as `alanfranz/fpm-within-docker`) ?

Reply:
I looked but couldn't find an existing image that pinned the fpm version and had the ability to build both deb and rpm packages.

Reply:
Looks like the one I mentioned does support both deb and rpm. It ues 1.11 right now but probably will upgrade without adding a new tag. I see we have custom images for local builds, may be we can add support for both deb and rpm to the local dockerfiles and publish to otel org, and then use that in CI (I can help publish)

Reply:
The local build has been updated to use a single image for both rpm and deb.

I think using a published image for CI will eventually be the way to go.  But for now, since the packaging is still under development and may require changes to the Dockerfile and then re-publishing the image, we can avoid these extra steps with just doing the setup in the circleci job.  Thoughts?

Reply:
Sounds good.